Makefile

```diff
@@ -445,39 +445,6 @@ bulk_data:
 	fi; \
 	$(PYTHON) tools/data_generators/rbac_dummy_data_generator.py --preset=$(DATA_GEN_PRESET)
 
-# l10n TASKS
```

Comment:
I am removing this `check-po` target because I can't find a place where we actually use it anymore.  It looks like it's purpose was to check the Language header in the .po files to make sure the valid locale for Chinese is set.  

Original commit: https://github.com/ansible/awx/commit/40b94a38a86aa0d2a4cc38861c3d1bd64e3feb3e